service/worker/scanner/shardscanner/aggregators.go

```diff
@@ -330,6 +330,17 @@ func (a *ShardFixResultAggregator) GetReport(shardID int) (*FixReport, error) {
 	return nil, fmt.Errorf("shard %v has not finished yet, check back later for report", shardID)
 }
 
+func (a *ShardFixResultAggregator) GetAllFixResults() (map[int]FixResult, error) {
+	result := make(map[int]FixResult, len(a.reports))
+	for k, v := range a.reports {
+		if v.Result.Empty() {
+			continue
+		}
```

Comment:
all shards have reports, so in the normal / healthy case this was producing a pile of 
```
"1": {
  "ShardFixKeys": {
    "Fixed": null,
    "Skipped": null,
    "Failed": null,
  }
  "ControlFlowFailure": null,
}
```
which is a lot of unnecessary output.

a possibly-nicer way to do this would be to use json tags and mark these as `omitempty`, but I'm not sure what things might be depending on the exact serialized structure at the moment, so I'm avoiding that.